### DISCUSSION TITLE
Collabora error - Unable to start container

### DISCUSSION THREAD

**Discussion by vicky-bs** [nextcloud/all-in-one#3605] [Questions] [answered]
### Steps to reproduce
1. Install fresh copy of nextcloud aio
2. Select collabora in admin page

### Actual behavior 

Collabora logs as below. container shows it's healthy

Setting up watches.
Watches established.
/etc/coolwsd/proof_key exists already. New proof key was not generated.
wsd-00006-00006 2023-10-24 23:13:23.900186 +0530 [ coolwsd ] INF  Initializing wsd. Local time: Tue 2023-10-24 23:13:23 +0530. Log level is [8].| common/Log.cpp:330
wsd-00006-00006 2023-10-24 23:13:23.900246 +0530 [ coolwsd ] INF  Setting log-level to [trace] and delaying setting to [warning] until after WSD initialization.| wsd/COOLWSD.cpp:2207
wsd-00006-00006 2023-10-24 23:13:23.900261 +0530 [ coolwsd ] INF  Initializing coolwsd server [nextcloud.vickybs.in]. Experimental features are enabled.| wsd/COOLWSD.cpp:2213
wsd-00006-00006 2023-10-24 23:13:23.900285 +0530 [ coolwsd ] DBG  New SocketPoll [UnitWSD] owned by 0x7f9f5898b840| net/Socket.cpp:193
wsd-00006-00006 2023-10-24 23:13:23.900396 +0530 [ coolwsd ] INF  Anonymization of user-data is configurable.| wsd/COOLWSD.cpp:2262
wsd-00006-00006 2023-10-24 23:13:23.900421 +0530 [ coolwsd ] WRN  NOTE: both logging.anonymize.usernames and logging.anonymize.filenames are deprecated and superseded by logging.anonymize.anonymize_user_data. Please remove username and filename entries from the config and use only anonymize_user_data.| wsd/COOLWSD.cpp:2273
wsd-00006-00006 2023-10-24 23:13:23.900429 +0530 [ coolwsd ] WRN  Since logging.anonymize.anonymize_user_data is provided (false) in the config, it will be used.| wsd/COOLWSD.cpp:2276
wsd-00006-00006 2023-10-24 23:13:23.900437 +0530 [ coolwsd ] INF  Anonymization of user-data is disabled.| wsd/COOLWSD.cpp:2309
wsd-00006-00006 2023-10-24 23:13:23.900470 +0530 [ coolwsd ] INF  SSL support: SSL is disabled.| wsd/COOLWSD.cpp:2353
wsd-00006-00006 2023-10-24 23:13:23.900479 +0530 [ coolwsd ] INF  SSL support: termination is enabled.| wsd/COOLWSD.cpp:2354
wsd-00006-00006 2023-10-24 23:13:23.900492 +0530 [ coolwsd ] DBG  Setting envar PDFIMPORT_RESOLUTION_DPI=96 per config per_document.pdf_resolution_dpi| wsd/COOLWSD.cpp:2380
wsd-00006-00006 2023-10-24 23:13:23.900528 +0530 [ coolwsd ] INF  Creating childroot: /opt/cool/child-roots/6-4d0ac112/| wsd/COOLWSD.cpp:2420
wsd-00006-00006 2023-10-24 23:13:23.901095 +0530 [ coolwsd ] INF  Cleaning up childroot directory [/opt/cool/child-roots/].| common/JailUtil.cpp:196
wsd-00006-00006 2023-10-24 23:13:23.901174 +0530 [ coolwsd ] TRC  Directory [/opt/cool/child-roots/] is not a jail directory or doesn't exist.| common/JailUtil.cpp:201
wsd-00006-00006 2023-10-24 23:13:23.901190 +0530 [ coolwsd ] INF  Cleaning up childroot directory [/opt/cool/child-roots/6-4d0ac112/].| common/JailUtil.cpp:196
wsd-00006-00006 2023-10-24 23:13:23.901200 +0530 [ coolwsd ] TRC  Directory [/opt/cool/child-roots/6-4d0ac112/] is not a jail directory or doesn't exist.| common/JailUtil.cpp:201
wsd-00006-00006 2023-10-24 23:13:23.901207 +0530 [ coolwsd ] INF  Creating jail path (if missing): /opt/cool/child-roots/6-4d0ac112//tmp/incoming| common/JailUtil.cpp:275
Failed to initialize COOLWSD: File access error: Invalid argument: /opt/cool/child-roots
wsd-00006-00006 2023-10-24 23:13:23.901484 +0530 [ coolwsd ] FTL  Failed to initialize COOLWSD: File access error: Invalid argument: /opt/cool/child-roots| wsd/COOLWSD.hpp:509
File access error: Invalid argument: /opt/cool/child-roots

### Host OS 
Debian 12 LXC as privilege container on Proxmox

#### Nextcloud AIO version 
Nextcloud Hub 6  (27.1.2)
#### Current channel 
Stable
#### Other valuable info 
Multiple scenarios such as removing container, removing nextcloud aio and reinstalling doesn't help and gives the same error

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions?discussions_q=is%3Aopen+File+access+error%3A+Invalid+argument%3A+%2Fopt%2Fcool%2Fchild-root
